src/components/SkillFeedbackPage/SkillFeedbackPage.js

```diff
@@ -0,0 +1,575 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { Paper } from 'material-ui';
+import Cookies from 'universal-cookie';
+import PropTypes from 'prop-types';
+import $ from 'jquery';
+
+import StaticAppBar from '../StaticAppBar/StaticAppBar.react';
+import Footer from '../Footer/Footer.react';
+import { List, ListItem } from 'material-ui/List';
+import Divider from 'material-ui/Divider';
+import IconButton from 'material-ui/IconButton';
+import CircleImage from '../CircleImage/CircleImage';
+import Dialog from 'material-ui/Dialog';
+import FlatButton from 'material-ui/FlatButton';
+import RaisedButton from 'material-ui/RaisedButton';
+import TextField from 'material-ui/TextField';
+import IconMenu from 'material-ui/IconMenu';
+import MenuItem from 'material-ui/MenuItem';
+import CircularProgress from 'material-ui/CircularProgress';
+import AuthorSkills from '../AuthorSkills/AuthorSkills';
+
+import MoreVertIcon from 'material-ui/svg-icons/navigation/more-vert';
+import Delete from 'material-ui/svg-icons/action/delete';
+import EditBtn from 'material-ui/svg-icons/editor/mode-edit';
+import NavigationChevronRight from 'material-ui/svg-icons/navigation/chevron-right';
+
+import '../SkillFeedbackCard/SkillFeedbackCard.css';
+import './SkillFeedbackPage.css';
+import urls from '../../Utils/urls';
+
+import { parseDate } from '../../Utils/helperFunctions';
+
+const cookies = new Cookies();
+
+const defaultNullSkillList = ['image', 'author', 'author_url'];
+// eslint-disable-next-line
+let name;
+
+const iconButtonElement = (
+  <IconButton touch={true} tooltip="More" tooltipPosition="bottom-left">
+    <MoreVertIcon />
+  </IconButton>
+);
+
+class SkillFeedbackPage extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      openEditDialog: false,
+      openDeleteDialog: false,
+      errorText: '',
+      skill_feedback: [],
+      imgUrl: null,
+      dataReceived: false,
+      showAuthorSkills: false,
+    };
+    let clickedSkill = this.props.location.pathname.split('/')[2];
+    this.name = clickedSkill;
+    this.url = urls.API_URL + '/cms/getSkillList.json?group=Knowledge';
```

Comment:
Is it used anywhere?

Reply:
Yes, it is used

Reply:
Ah okay then, I don't see the need to fetch all skills data in the component though or is it something that I might've missed?

Reply:
No, you are correct, I have updated the code. Thanks!